site/gatsby-site/src/templates/cite.js

```diff
@@ -181,7 +181,7 @@ function CitePage(props) {
                   <Button
                     variant="outline-primary"
                     className="me-2"
-                    href={'/incidents/edit?incident_id=' + incident.incident_id}
+                    href={'/incidents/edit?incidentId=' + incident.incident_id}
```

Comment:
Parameters should mirror their expression in the field definition. This should remain `incident_id` and this should be `const [incidentId] = useQueryParam('incident_id', withDefault(NumberParam, 1));` no?

Reply:
Sure, it is my js mind that doesn't like snake_case, updating.

Reply:
https://github.com/responsible-ai-collaborative/aiid/issues/554